packages/Button/stories/Button.stories.js

```diff
@@ -2,15 +2,15 @@ import React from "react";
 import { storiesOf } from "@storybook/react";
 import { withKnobs } from "@storybook/addon-knobs";
 
-import Showcase from "./examples/Showcase";
+import Showcase, { showcaseProps } from "./examples/Showcase";
 import Basic from "./examples/Basic";
 import NewRef from "./examples/NewRef";
 import OldRef from "./examples/OldRef";
 import CloseButtonRef from "./examples/CloseButtonRef";
 
 storiesOf("Button", module)
   .addDecorator(withKnobs)
-  .add("Showcase", () => <Showcase />)
+  .add("Showcase", () => <Showcase {...showcaseProps()} />)
```

Comment:
I may be missing something - but both `Showcase` and `showcaseProps` are exported from the same file. Why not have that file export just the component that renders the story with the props baked in?

Reply:
I may also be missing something, but this was the simplest way I could find to make the knobs work again (they had all stopped).  It seemed like the knob functions needed to be called outside of the example component...?? But I am not really sure _why_ they stopped working in this new version.